clients/appointments/src/components/index.tsx

```diff
@@ -53,8 +54,9 @@ export const AppointmentsView = ({
   }
 
   const handleError: HandleErrorType = (error, msg) => {
-    callbacks?.onError(error, msg)
+    callbacks?.onError?.(error, msg)
     setError(msg)
+    setLoading(false)
```

Comment:
This fixes the "infinite spinner" a user gets when certain network fails (cancelling an appointment as an example)

Reply:
Good addition